tests/main/services-start/task.yaml

```diff
@@ -0,0 +1,49 @@
+summary: Check that "snap start <snap>" affects only enabled services
+
+environment:
+    EXTRA_FLAGS/start: ""
+    EXPECTED_STATUS: "inactive"
+    STATUS_MSG: "not running"
+    EXTRA_FLAGS/enable: "--enable"
+    EXPECTED_STATUS/enable: "running"
+    STATUS_MSG/enable: "running"
+
+execute: |
+    echo "Installing the service snap"
+
+    # we are using systemd-notify indicate the service is active, this is
+    # currently not allowed by daemon-notify interface, so we may as well just
+    # install in devmode
+    "$TESTSTOOLS"/snaps-state install-local test-snapd-service-restart --devmode
+
+    echo "Stop all services"
+    snap stop test-snapd-service-restart
+
+    echo "Verify that they are all stopped"
+    for id in 1 2 3 4; do
+        systemctl status snap.test-snapd-service-restart.svc${id}.service | MATCH "inactive"
```

Comment:
this could be `snap services test-snapd-service-restart.svc${id}`

Reply:
This is prefer to keep it as it is, to be 100% sure that the outcome is the desired one even in case `snap services` has some bugs. Or I can _add_ this, if you think it's worth.

Reply:
this is fine as-is thanks